Marlin/ultralcd.cpp

```diff
@@ -64,6 +64,9 @@ void lcd_status_screen();
 millis_t next_lcd_update_ms;
 
 uint8_t lcdDrawUpdate = LCDVIEW_CLEAR_CALL_REDRAW; // Set when the LCD needs to draw, decrements after every draw. Set to 2 in LCD routines so the LCD gets at least 1 full redraw (first redraw is partial)
+#if ENABLED(DOGLCD)
+  uint8_t drawing_screen = 0;
```

Comment:
Could be bool, but `u8g.nextPage()` returns something like a uint8_t.
bool and uint8_t have the same size and the same speed - why cast?